packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/ios/Podfile

```diff
@@ -41,8 +41,12 @@ end
 post_install do |installer|
   installer.pods_project.targets.each do |target|
     flutter_additional_ios_build_settings(target)
-    target.build_configurations.each do |config|
```

Comment:
This line should not be deleted - I assume you removed it by accident 😄

Reply:
Yes 😅